motor/core.py

```diff
@@ -1795,3 +1796,31 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         pass
+
+class AgnosticClientEncryption(AgnosticBase):
+    __motor_class_name__ = 'MotorClientEncryption'
+    __delegate_class__ = ClientEncryption
+
+    create_data_key = AsyncCommand()
+    encrypt = AsyncCommand()
+    decrypt = AsyncCommand()
+    close = AsyncCommand()
+
+    def __init__(self, kms_providers, key_vault_namespace, key_vault_client, codec_options, io_loop=None):
```

Comment:
Let's add a docstring here. You can take inspiration from AgnosticClient.

Reply:
I'll adapt the one from the PyMongo class